control/controlclient/direct.go

```diff
@@ -1401,7 +1403,7 @@ func answerHeadPing(logf logger.Logf, c *http.Client, pr *tailcfg.PingRequest) {
 	if pr.Log {
 		logf("answerHeadPing: sending HEAD ping to %v ...", pr.URL)
 	}
-	t0 := time.Now()
+	t0 := clockNow()
 	_, err = c.Do(req)
 	d := time.Since(t0).Round(time.Millisecond)
```

Comment:
these `time.Since` calls need to be converted for it to make sense

This is a pattern that appears often enough that I wonder if we should have something like `tstime.Stopwatch` that can handle this in a way that requires less plumbing and remains easy to use. This is mostly an observation, not a recommendation to make any changes, unless you have any good ideas for how to implement such a thing easily. Maybe one approach would be to not bother simulating time for "action took X ms" use cases if they're just used for logging purposes. Going through a `tstime` wrapper would still keep the rules about stdlib `time` usage clear and make static checks straightfoward.